### ISSUE TITLE
For better readibility, I would display or hide the AIL 3&4, FLAP 3&4 depending of the use in the various mixers

### ISSUE THREAD

**Issue by prondel** [FrSky-OS/discussion#16]
99% of the sailplane uses only 4 wing servos. Here in the wing mixer there is alway 4 ailerons servos and 4 flaps servos.

It would be good to see only them when configured.

**Comment by AloftHobbies** [comment]
I agree. When setting up the basic model parameters (ie glider) we can define how many flap and aileron channels are needed. It should default to 2 of each. I personally have been flying for 30 plus years and have not yet had a plane with more than 2 aileron or 2 flap channels.
